package proto_ff;

import "yd_fieldoptions.proto";


message E_WaremblemWaremblemSkillDesc
{
	optional int32 m_star = 1[(yd_fieldoptions.field_cname) = "解锁星级"];
	optional int32 m_id = 2[(yd_fieldoptions.field_cname) = "id"];
}

message E_WaremblemWaremblem
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 m_order = 3[(yd_fieldoptions.field_cname) = "解锁顺序"];
	optional int32 m_lvIndex = 4[(yd_fieldoptions.field_cname) = "lv索引"];
	optional int32 m_starIndex = 5[(yd_fieldoptions.field_cname) = "星级索引"];
	repeated E_WaremblemWaremblemSkillDesc m_skill = 6[(yd_fieldoptions.field_cname) = "技能", (yd_fieldoptions.field_arysize) = 5];
}


message Sheet_WaremblemWaremblem
{
	repeated E_WaremblemWaremblem E_WaremblemWaremblem_List = 1[(yd_fieldoptions.field_arysize)=64];
}

message E_WaremblemLvAttributeDesc
{
	optional int32 m_value = 1[(yd_fieldoptions.field_cname) = "值"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "ID"];
}

message E_WaremblemLv
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_index = 2[(yd_fieldoptions.field_cname) = "索引"];
	optional int32 m_lv = 3[(yd_fieldoptions.field_cname) = "lv"];
	optional int32 m_classBonus = 4[(yd_fieldoptions.field_cname) = "阶级加成"];
	optional int32 m_costId = 5[(yd_fieldoptions.field_cname) = "消耗道具ID"];
	optional int32 m_costNum = 6[(yd_fieldoptions.field_cname) = "消耗数量"];
	repeated E_WaremblemLvAttributeDesc m_attribute = 7[(yd_fieldoptions.field_cname) = "基础属性", (yd_fieldoptions.field_arysize) = 4];
}


message Sheet_WaremblemLv
{
	repeated E_WaremblemLv E_WaremblemLv_List = 1[(yd_fieldoptions.field_arysize)=512];
}

message E_WaremblemStarAttributeDesc
{
	optional int32 m_value = 1[(yd_fieldoptions.field_cname) = "值"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "ID"];
}

message E_WaremblemStar
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_index = 2[(yd_fieldoptions.field_cname) = "索引"];
	optional int32 m_star = 3[(yd_fieldoptions.field_cname) = "星级"];
	optional int32 m_costId = 4[(yd_fieldoptions.field_cname) = "消耗道具"];
	optional int32 m_costNum = 5[(yd_fieldoptions.field_cname) = "消耗数量"];
	optional int32 m_succes = 6[(yd_fieldoptions.field_cname) = "成功率"];
	repeated E_WaremblemStarAttributeDesc m_attribute = 7[(yd_fieldoptions.field_cname) = "基础属性", (yd_fieldoptions.field_arysize) = 5];
}


message Sheet_WaremblemStar
{
	repeated E_WaremblemStar E_WaremblemStar_List = 1[(yd_fieldoptions.field_arysize)=256];
}
